### PR TITLE
Add Pulsar Community Weekly Update - 2020-08-29~ 2020-09-04

### PR DESCRIPTION
**Pull request by Huanli-Meng**

### CODE REVIEW

File: content/2020/2020-09/2020-09-04-pulsar-weekly.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+- [Transaction] Handle acknowledgement in the transaction.
+
+    https://github.com/apache/pulsar/pull/7856 (@[gaoran10](https://github.com/gaoran10))
+
+- [Transaction] Support aborting transaction on partitions.
+
+    https://github.com/apache/pulsar/pull/7953 (@[gaoran10](https://github.com/gaoran10))
+
+- [Transaction] Optimize the logic for consume transaction messages.
```
Comment (Jennifer88huang) [inline content/2020/2020-09/2020-09-04-pulsar-weekly.md:23]:
```suggestion
- [Transaction] Optimize the logic for consuming transaction messages.
```

File: content/2020/2020-09/2020-09-04-pulsar-weekly.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+    https://github.com/apache/pulsar/pull/7833 (@[gaoran10](https://github.com/gaoran10))
+
+- [Topic Policy] Support topic-level `inactiveTopicPolicies`.
```
Comment (Jennifer88huang) [inline content/2020/2020-09/2020-09-04-pulsar-weekly.md:27]: can we improve the PR title?

File: content/2020/2020-09/2020-09-04-pulsar-weekly.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+    https://github.com/apache/pulsar/pull/7881 (@[hangc0276](https://github.com/hangc0276))
+
+### Notable Bug Fix
+
+- [Broker] Fix NPE when acknowledging messages at the broker side.
+
+    https://github.com/apache/pulsar/pull/7937 (@[Technoboy-](https://github.com/Technoboy-))
+
+- [Broker] Stop dispatching messages when messages are skipped temporally since the Key_Shared consumer is stuck on delivery.
```
Comment (Jennifer88huang) [inline content/2020/2020-09/2020-09-04-pulsar-weekly.md:49]: temporarily?

**Comment by Huanli-Meng** [comment]
updated. @Jennifer88huang . PTAL

**Review by Jennifer88huang — APPROVED** [review]

**Pull request merged**